checks/tasks/mail.py

```diff
@@ -824,3 +849,76 @@ def dmarc_get_public_suffix_list():
             public_suffix_list = dmarc_get_public_suffix_list()
 
     return public_suffix_list
+
+
+def tlsrpt_callback(data, status, r):
```

Comment:
Can we add types for these parameters? We try to do it for all new code. I have no idea what "status" means here, or why it should be 0.